src/Microsoft.Diagnostics.Monitoring.WebApi/Controllers/ExceptionsController.cs

```diff
@@ -21,43 +23,50 @@ namespace Microsoft.Diagnostics.Monitoring.WebApi.Controllers
     [Authorize(Policy = AuthConstants.PolicyName)]
     [ProducesErrorResponseType(typeof(ValidationProblemDetails))]
     [ProducesResponseType(typeof(ValidationProblemDetails), StatusCodes.Status400BadRequest)]
-    public sealed class ExceptionsController : ControllerBase
+    public sealed class ExceptionsController :
+        DiagnosticsControllerBase
     {
-        private readonly IExceptionsStore _exceptionsStore;
-        private readonly IOptions<ExceptionsOptions> _exceptionsOptions;
-        private readonly IExceptionsOperationFactory _operationFactory;
+        private readonly IOptions<ExceptionsOptions> _options;
 
-        public ExceptionsController(IServiceProvider serviceProvider)
+        public ExceptionsController(
+            IServiceProvider serviceProvider,
+            ILogger<ExceptionsController> logger)
+            : base(serviceProvider.GetRequiredService<IDiagnosticServices>(), logger)
         {
-            // The exceptions store for the default process
-            _exceptionsStore = serviceProvider.GetRequiredService<IExceptionsStore>();
-            _exceptionsOptions = serviceProvider.GetRequiredService<IOptions<ExceptionsOptions>>();
-            _operationFactory = serviceProvider.GetRequiredService<IExceptionsOperationFactory>();
+            _options = serviceProvider.GetRequiredService<IOptions<ExceptionsOptions>>();
         }
 
         /// <summary>
-        /// Gets the exceptions from the default process.
+        /// Gets the exceptions from the target process.
         /// </summary>
```

Comment:
nit: Do we want to add the `param name` comments here since we use that everywhere else when we have a pid/uid/name option?